interfaces/default/js/newznab.js

```diff
@@ -71,7 +71,54 @@ function search(query, catid, indexer) {
                         });
                         item.attr = attributes;
 
+			// First, assign download client category based on each individual usenet indexer
                         var clean_category = item.category.split(' > ')[0].toLowerCase().trim()
+			// Second, try to match with the newznab default to make it uniform across all indexers
+			// Maincategories
+			// 1 = console
+			// 2 = movies
+			// 3 = audio
+			// 4 = pc
+			// 5 = tv
+			// 6 = xxx
+			// 7 = other
+			var main_category = item.attr['category'].split("0",1)
+			if(main_category == 1) {
+			    var clean_category = "console"
+			}
+			else if(main_category == 2) {
+			    var clean_category = "movies"
+			}
+			else if(main_category == 3) {
+			    var clean_category = "audio"
+			}
+			else if(main_category == 4) {
+			    var clean_category = "pc"
+			}
+			else if(main_category == 5) {
+			    var clean_category = "tv"
+			}
+			else if(main_category == 6) {
+			    var clean_category = "xxx"
+			}
+			else if(main_category == 7) {
+			    var clean_category = "other"
+			}
+			else {
+			    var clean_category
+			}
+			// Subcategories
+			// 7020 = ebook
+			// 7030 = comics
+			if(item.attr['category'] == 7020) {
+			    var clean_category = "ebook"
+			}
+			else if(item.attr['category'] == 7030) {
+			    var clean_category = "comics"
+			}
+			else {
+			    var clean_category
```

Comment:
Thanks for the feedback. I wanted to make clean_category equal to:
https://github.com/kooimens/HTPC-Manager/blob/patch-7/interfaces/default/js/newznab.js#L75
if not matches from the manual set categories, but I now understand I can just leave it like it is in de new commit:)